.gitlab/corona-jobs.yml

```diff
@@ -4,12 +4,12 @@
 #
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
-rocm_4_1_fortran_gcc_8_1_0 (build and test on corona):
+rocm_4_5_2_fortran_clang_13_0_0 (build and test on corona):
```

Comment:
Why did we switch from gcc_8_1_0 to clang_13_0_0?

Reply:
Documenting what we talked about in dev meeting: GCC on corona only works for hip_runtime, not hip, so for now we have to use amdclang (which is clang 13) and hopefully at some point that will extend to cce